packages/manager/apps/public-cloud/src/components/walkMe/templates/popover.html

```diff
@@ -0,0 +1,14 @@
+<div class="popover tour oui-popover visible">
+    <div class="walkme-line"></div>
+    <button class="oui-button oui-button_icon-alone walkme-close mr-1 pt-2 position-absolute" data-role="end">
```

Comment:
is this supposed to be `ARIA` ?

because `role="end"` doesn't exists: https://www.w3.org/TR/wai-aria-1.1/#end

:)